### DISCUSSION TITLE
v19.4.0

### DISCUSSION THREAD

**Discussion by ghost** [uNetworking/uWebSockets#1319] [General]
# Stampede tweaks
* Tweaks the TLS handshake queue to better cope with mass connections and mass disconnections.
* Adjusts the WebSocket shutdown timeout from a default of 120 seconds to roughly 4 seconds, making mass disconnections finish quicker.

Please report any issues or undesired side effects introduced in this release.

<hr /><em>This discussion was created from the release <a href='https://github.com/uNetworking/uWebSockets/releases/tag/v19.4.0'>v19.4.0</a>.</em>

**Comment by ghost** [comment]
@not-implemented Your changes are in this release.

**Reply by not-implemented** [reply]
Cool, thanks!
